clang/runtime/dpct-rt/include/dpl_extras/algorithm.h.inc

```diff
@@ -1774,13 +1774,17 @@ inline void segmented_sort_pairs(
     _ExecutionPolicy &&policy, io_iterator_pair<key_t> &keys,
     io_iterator_pair<value_t> &values, int64_t n, int64_t nsegments,
     OffsetIteratorT begin_offsets, OffsetIteratorT end_offsets,
-    bool descending = false, int begin_bit = 0,
+    bool descending = false, bool do_swap_iters = false, int begin_bit = 0,
```

Comment:
Seems a ill-formed syntax, please define variables in sperate line.

Reply:
These are all default parameters in a parameter list, so I'm not sure the problem here.

Reply:
Ah, my mistake.